data/campaigns/fri02.wmf/scripting/texts.lua

```diff
@@ -124,22 +124,30 @@ obj_defeat_both = {
 -- Texts to the user
 -- ==================
 
-campaign_data_warning = {
-   title = _"Warning",
-   body = p(_[[You appear not to have completed the previous mission. You may still play this scenario, but you will be at a disadvantage. I recommend that you complete the previous scenario first and then restart this mission.]]),
-   w = 450,
-   h = 150,
-}
+-- TRANSLATORS: Reebaud – Introduction 1
+local intro_1_start = _([[I began to believe the journey would never end. I have no idea how long we were on the sea, tossed about by envious storms… it must have been months.]])
+-- TRANSLATORS: Reebaud – Introduction 1
+local intro_1_iron = _([[The storms have maintained a firm grip on the ship throughout our voyage. There was hardly a day on which we did not wonder whether it would be our last. We were compelled to drop the precious metals we have taken with us overboard to steady the ship, feeling as though it were a squalid sacrifice to the gods to give us hope to live a few hours longer.]])
```

Comment:
```suggestion
local intro_1_iron = _([[The storms maintained a firm grip on the ship throughout our voyage. There was hardly a day on which we did not wonder whether it would be our last. We were compelled to drop the precious metals we had taken with us overboard to steady the ship, feeling as though it were a squalid sacrifice to the gods to give us hope to live a few hours longer.]])
```